examples/mqtt-client-publish-message/mqtt_client_publish_message.metatags

```diff
@@ -0,0 +1,2 @@
+description: BBE on sending messages to a MQTT topic using a `mqtt:Client`.
```

Comment:
```suggestion
description: BBE on sending messages to a MQTT topic using a `mqtt:Client`.
```
Ditto

Reply:
This is not a prerequisite as other 2 steps